### ISSUE TITLE
WinRM over HTTPS

### ISSUE THREAD

**Issue by TraGicCode** [puppetlabs/bolt#122]
I cannot seem to connect to my remote machines using winrm /w https.  I assume this is not currently implemented.  The below are the commands i have tried.

```powershell
bolt command run 'get-childitem' --nodes winrm://ZZZZZ.XXXX.com:5986
```

```powershell
bolt command run 'get-childitem' --nodes winrm://ZZZZZ.XXXX.com
```

**Comment by joshcooper** [comment]
Hi @TraGicCode the underlying `winrm` library supports it, but the options for enabling it are not exposed in Bolt yet. We're currently tracking this in https://tickets.puppetlabs.com/browse/BOLT-125.

**Comment by MikaelSmith** [comment]
This should be available in the upcoming Bolt 0.12.0 release.
